<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TvSeasonNewDialog</class>
 <widget class="QDialog" name="TvSeasonNewDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>118</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>新增部</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="hLayout_Name">
     <item>
      <widget class="QLabel" name="label_Name">
       <property name="text">
        <string>名称</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_Name"/>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_Display">
       <property name="styleSheet">
        <string notr="true">QCheckBox
{
  spacing: 0px;
}
QCheckBox::indicator
{
  width: 20px;
  height: 20px;
}
QCheckBox::indicator:checked:enabled
{
  image:url(:/File/img/box_check_enable.png);
}
QCheckBox::indicator:unchecked:enabled
{
  image:url(:/File/img/box_uncheck_enable.png);
}
QCheckBox::indicator:unchecked:disabled
{
  image:url(:/File/img/box_uncheck_disable.png);
}</string>
       </property>
       <property name="text">
        <string>公开</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hLayout_ReleaseDate">
     <item>
      <widget class="QLabel" name="label_ReleaseDate">
       <property name="text">
        <string>发布日期</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_ReleaseDateEnable">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QCheckBox
{
  spacing: 0px;
}
QCheckBox::indicator
{
  width: 20px;
  height: 20px;
}
QCheckBox::indicator:checked:enabled
{
  image:url(:/File/img/box_check_enable.png);
}
QCheckBox::indicator:unchecked:enabled
{
  image:url(:/File/img/box_uncheck_enable.png);
}
QCheckBox::indicator:unchecked:disabled
{
  image:url(:/File/img/box_uncheck_disable.png);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dateEdit_ReleaseDate">
       <property name="displayFormat">
        <string notr="true">yyyy-MM-dd</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_Point">
       <property name="text">
        <string>评分</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_Point">
       <property name="maxVisibleItems">
        <number>13</number>
       </property>
       <item>
        <property name="text">
         <string>未评 0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>烂作 1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>不行 2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>极差 3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>很差 4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>略差 5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>及格 6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>海星 7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>一般 8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>很好 9</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>完美 10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>极好 11</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>神作 12</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_CollectIt">
       <property name="styleSheet">
        <string notr="true">QCheckBox
{
  spacing: 0px;
}
QCheckBox::indicator
{
  width: 20px;
  height: 20px;
}
QCheckBox::indicator:checked:enabled
{
  image:url(:/File/img/box_check_enable.png);
}
QCheckBox::indicator:unchecked:enabled
{
  image:url(:/File/img/box_uncheck_enable.png);
}
QCheckBox::indicator:unchecked:disabled
{
  image:url(:/File/img/box_uncheck_disable.png);
}</string>
       </property>
       <property name="text">
        <string>要收藏</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_CollectOk">
       <property name="styleSheet">
        <string notr="true">QCheckBox
{
  spacing: 0px;
}
QCheckBox::indicator
{
  width: 20px;
  height: 20px;
}
QCheckBox::indicator:checked:enabled
{
  image:url(:/File/img/box_check_enable.png);
}
QCheckBox::indicator:unchecked:enabled
{
  image:url(:/File/img/box_uncheck_enable.png);
}
QCheckBox::indicator:unchecked:disabled
{
  image:url(:/File/img/box_uncheck_disable.png);
}</string>
       </property>
       <property name="text">
        <string>已收藏</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="hSpacer_Collect">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="vSpacer_Space">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hLayout_OkCancel">
     <item>
      <spacer name="hSpacer_OkCancel">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Ok">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Cancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
